trace/trace_quickstart/main.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2017 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// [START trace_quickstart]
+// Sample trace_quickstart creates traces incoming and outgoing requests.
+package main
+
+import (
+	"log"
+	"net/http"
+
+	// Imports the Google Cloud Trace client package.
+	"cloud.google.com/go/trace"
+	"golang.org/x/net/context"
+)
+
+func main() {
+	ctx := context.Background()
+
+	// Sets your Google Cloud Platform project ID.
+	projectID := "YOUR_PROJECT_ID"
+
+	// Creates a client.
+	traceClient, err := trace.NewClient(ctx, projectID)
+	if err != nil {
+		log.Fatalf("Failed to create client: %v", err)
+	}
+
+	httpClient := http.Client{
+		Transport: &trace.Transport{},
+	}
+
+	handler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		req, _ := http.NewRequest("GET", "https://metadata/users", nil)
+		req = req.WithContext(r.Context())
```

Comment:
A comment for this line, perhaps? This is the part that propagates the Trace ID, right?